fastlane/download_metadata.swift

```diff
@@ -133,3 +156,11 @@ func downloadTranslation(languageCode: String, folderName: String) {
 languages.forEach { (key: String, value: String) in
     downloadTranslation(languageCode: value, folderName: key)
 }
+
+extension Array where Element == String {
+
+    var second: String? {
+        guard count >= 2 else { return .none }
+        return self[1]
+    }
+}
```

Comment:
nit (I too want to have some Swift fun! 😄 )

```suggestion
extension Array {
    var second: Element? { dropFirst().first }
}
```